Sources/Conformance/main.swift

```diff
@@ -120,6 +120,8 @@ func singleTest() throws -> Bool {
 }
 
 Google_Protobuf_Any.register(messageType: Conformance_TestAllTypes.self)
+Google_Protobuf_Any.register(messageType: ProtobufTestMessages_Proto3_TestAllTypes.self)
```

Comment:
That assumes you are using descriptors to translate JSON to protobuf, which in turn relies on a registry of descriptors.